fastlane/Fastfile

```diff
@@ -64,7 +64,7 @@ end
 platform :android do
 
   before_all do
-    apk_file_path = Dir["#{ENV["CIRCLE_ARTIFACTS"]}/*.apk"].first
+    apk_file_path = Dir["/home/circleci/code/cordova/platforms/android/app/build/outputs/apk/*/*.apk"].first
```

Comment:
is the `CIRCLE_ARTIFACTS` variable gone ? 😱

Reply:
oh sorry I missed that part of the description
Can't we replace it in the env instead of the Fastfile

Reply:
yes better to update in CIRCLE CI ENV

Reply:
We can add our own environment variable for this so we can move it from fastfile. :)